src/lib/ui/app/InsightCard/Tags.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+  import Tag from './Tag.svelte'
+
+  const { items } = $props()
+</script>
+
+<div class="gap-1.5 row">
+  {#each items as { name }}
+    <Tag {name} />
```

Comment:
I would say this component is too small to divide in to multiple components, but that’s a nitpick

Reply:
Agreed. I removed `Tag` component and move it here.